git-add--interactive.perl

```diff
@@ -1717,16 +1717,7 @@ sub quit_cmd {
 }
 
 sub help_cmd {
-# TRANSLATORS: please do not translate the command names
-# 'status', 'update', 'revert', etc.
```

Comment:
Yep, this suggests that the funny `__` below means that the text is translatable. And we should also copy the comment, like it is done elsewhere (the comment should *not* wrap after the initial `/*`, see e.g. https://github.com/git/git/blob/v2.20.1/sequencer.c#L1793-L1794